src/components/WeatherWidget/WeatherWidgetIcon.test.tsx

```diff
@@ -0,0 +1,485 @@
+        {}
+      )
+    })
+
+    await waitFor(() => {
+      expect(screen.getByLabelText('Icon for smog')).toBeInTheDocument()
+    })
+  })
+
+  test('renders the WeatherWidgetIcon component with a temperature-high icon when temp is at least 80', async () => {
+    const mockHourlyPeriod = {
+      windSpeed: 0,
+      isDaytime: true,
+      shortForecast: '',
+      temperature: 80,
+    }
+
+    await act(async () => {
+      renderWithMySpaceAndModalContext(
+        <WeatherWidgetIcon hourlyPeriod={mockHourlyPeriod} iconSize="2xl" />,
+        {}
+      )
+    })
+
+    await waitFor(() => {
+      expect(
+        screen.getByLabelText('Icon for temperature-high')
+      ).toBeInTheDocument()
+    })
+  })
+
+  test('renders the WeatherWidgetIcon component with a temperature-low icon when temp is 60 or less', async () => {
+    const mockHourlyPeriod = {
+      windSpeed: 0,
+      isDaytime: true,
+      shortForecast: '',
+      temperature: 59,
+    }
+
+    await act(async () => {
+      renderWithMySpaceAndModalContext(
+        <WeatherWidgetIcon hourlyPeriod={mockHourlyPeriod} iconSize="2xl" />,
+        {}
+      )
+    })
+
+    await waitFor(() => {
+      expect(
+        screen.getByLabelText('Icon for temperature-low')
+      ).toBeInTheDocument()
+    })
+  })
+
+  test('renders the WeatherWidgetIcon component with a sun icon when shortForecast is empty/unknown', async () => {
+    const mockHourlyPeriod = {
+      windSpeed: 0,
+      isDaytime: true,
+      shortForecast: '',
+      temperature: 70,
+    }
+
+    await act(async () => {
+      renderWithMySpaceAndModalContext(
+        <WeatherWidgetIcon hourlyPeriod={mockHourlyPeriod} />,
+        {}
+      )
+    })
+
+    await waitFor(() => {
+      expect(screen.getByLabelText('Icon for sun')).toBeInTheDocument()
+    })
+  })
+
+  test('renders the WeatherWidgetIcon component with a moon icon when isDaytime is false', async () => {
+    const mockHourlyPeriod = {
+      windSpeed: 0,
+      isDaytime: false,
+      shortForecast: 'Clear',
+      temperature: 70,
+    }
+
+    await act(async () => {
+      renderWithMySpaceAndModalContext(
+        <WeatherWidgetIcon hourlyPeriod={mockHourlyPeriod} iconSize="2xl" />,
+        {}
+      )
+    })
+
+    await waitFor(() => {
+      expect(screen.getByLabelText('Icon for moon')).toBeInTheDocument()
+    })
+  })
+
+  test('renders the WeatherWidgetIcon component with a cloud icon when isDaytime is false and shortForecast includes "cloudy"', async () => {
+    const mockHourlyPeriod = {
+      windSpeed: 0,
+      isDaytime: false,
+      shortForecast: 'Cloudy',
+      temperature: 70,
+    }
+
+    await act(async () => {
+      renderWithMySpaceAndModalContext(
+        <WeatherWidgetIcon hourlyPeriod={mockHourlyPeriod} iconSize="2xl" />,
+        {}
+      )
+    })
+
+    await waitFor(() => {
+      expect(screen.getByLabelText('Icon for cloud')).toBeInTheDocument()
```

Comment:
Question: are there different icons for cloudy or rain that are different for night or day? I don't think so but wanted to confirm.

Reply:
In font awesome? Not that I'm aware of. But importing into the app? No, we're using the same ones.